osbenchmark/benchmark.py

```diff
@@ -176,6 +182,10 @@ def add_workload_source(subparser):
         "--output-path",
         default=os.path.join(os.getcwd(), "workloads"),
         help="Workload output directory (default: workloads/)")
+    create_workload_parser.add_argument(
+        "--total-docs",
+        type=list_or_none,
```

Comment:
May be better to just specify `opts.csv_to_list` here, rather than creating another function.